accord-core/src/main/java/accord/api/Key.java

```diff
@@ -3,10 +3,7 @@
 /**
  * A routing key for determining which shards are involved in a transaction
  */
-public interface Key<K extends Key<K>> extends Comparable<K>
+public interface Key<K extends Key<K>> extends RoutingKey<K>
 {
-    /**
-     * Returns a hash code of a key to support accord internal sharding. Hash values for equal keys must be equal.
-     */
-    int keyHash();
+    Key toRoutingKey();
```

Comment:
> toRoutingKey appears to be dead in this patch set yes, I'll remove it.

just reposting so it doesn't get lost, seems after the last update It doesn't show up while reading the code.

Reply:
Last update was just a rebase to get rid of the conflicts, I haven't addressed your comments yet - sorry for any confusion!

Reply:
no confusion at all!  I personally rely on GH comments to make sure nothing gets dropped, so when GH starts to hide things I like to re-comment just to make sure it doesn't get lost.

Reply:
I do the same, I just rely on the 'resolved' indicator, and make sure to expand the 'Outdated' ones